deploy/infrastructure/modules/terraform-google-dss/TFVARS.md

```diff
@@ -119,7 +120,7 @@ of this .pem file here as /public-certs/YOUR-KEY-NAME.pem replacing YOUR-KEY-NAM
 if using the provided us-demo.pem, use the path /public-certs/us-demo.pem. Note that your .pem file should built
 in the docker image or mounted manually.
 Example:
-```json
+```
```

Comment:
nit: I'd recommend (substituting backticks where appropriate)

```
Example 1 (dummy auth):
'''json
{
  public_key_pem_path = "/test-certs/auth2.pem"
}
'''
Example 2:
'''json
{
  public_key_pem_path = "/jwt-public-certs/us-demo.pem"
}
'''
```

Reply:
Done

Reply:
They have to be backticks though (not apostrophes); I didn't put backticks in my recommendation because then the preview wouldn't have rendered correctly :)

Reply:
👍